doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -9,7 +10,7 @@ alter_index ::=
     | 'RESET' '(' field ( ',' field )* ')'
   )
 array_agg ::=
-  'array_agg' '(' values  ( 'ORDER' 'BY' col_ref ( 'ASC' | 'DESC' )? ( ',' col_ref ( 'ASC' | 'DESC' )? )* )? ')'
+  'array_agg' '(' values  ( 'ORDER' 'BY' col_ref ( 'ASC' | 'DESC' )? ( ',' col_ref ( 'ASC' | 'DESC' )? )* )? ')' ('FILTER' '(' 'WHERE' filter_clause ')')?
```

Comment:
Do we describe the syntax of `filter_clause` anywhere?

Reply:
we don't. PG does not go into any details on it, so I followed their example.